core/storage/classifier/gae_models.py

```diff
@@ -63,8 +58,10 @@ class ClassifierTrainingJobModel(base_models.BaseModel):
     # It is incremented by TTL when a job with status NEW is picked up by VM.
     next_scheduled_check_time = ndb.DateTimeProperty(
         required=True, indexed=True)
-    # The schema version for the data that is being classified.
-    data_schema_version = ndb.IntegerProperty(required=True, indexed=True)
+    # The algorithm version for the classifier. Algorithm version identifies
+    # the format of the classifier_data as well as the prediction API to be
+    # used.
+    algorithm_version = ndb.IntegerProperty(required=True, indexed=True)
```

Comment:
Do we need an one-off job to update this filed in the existing models? (OR we don't have any such models ion prod/test server?)

Reply:
There are no existing models.